src/coreclr/src/vm/diagnosticsprotocol.h

```diff
@@ -75,21 +75,10 @@ namespace DiagnosticsIpc
         Server        = 0xFF,
     };
 
-    // Overlaps with DiagnosticServerResponseId
-    // DON'T create overlapping values
-    enum class DiagnosticServerCommandId : uint8_t
-    {
-        // 0x00 used in DiagnosticServerResponseId
-        ResumeRuntime = 0x01,
```

Comment:
@jander-msft I'm moving the ResumeRuntime command to be under the `Process` command set being introduced in the PR to make things more semantically coherent.

Reply:
Thanks for the heads up. I've moved it to the Process command set in this PR.